ionic/components/button/button.md.scss

```diff
@@ -58,7 +58,7 @@ $button-md-small-icon-font-size:           1.4em !default;
               color $button-md-transition-duration $button-md-animation-curve;
 
   &:hover:not(.disable-hover) {
-    background-color: $button-md-clear-hover-background-color;
+    background-color: $button-md-color;
```

Comment:
the problem I see with that is that if the text color is black or any other dark color, with the `::after` pseudo selector the text will also be overlapped and it could look a bit lighter...

another alternative could be using `background-color: lighten($button-md-color, 10%)`